setup/providers/azure.md

```diff
@@ -7,6 +7,8 @@ sidebar:
 
 {% include toc %}
 
+If you want to configure Spinnaker to deploy containerized apps to Kubernetes on Azure, follow the steps [here](http://aka.ms/azspinlab). Otherwise, if you would like to deploy to Azure VM Scale Sets, follow the steps below.
```

Comment:
If you use an aka.ms link (as opposed to linking to the section directly) keep in mind it will be harder to maintain. For example, we will have to update the link when 'spinnaker.github.io' changes to 'spinnaker.io'